packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -473,6 +472,28 @@ export class Users {
       }
     );
 
+    const derivedDbMarketsInfos = await db.findMarkets({
+      selector: { market: { $in: marketIds } },
+    });
+    for (const derivedMarketDbInfo of derivedDbMarketsInfos) {
+      if (derivedMarketDbInfo.reportingState === MarketReportingState.Finalized || MarketReportingState.AwaitingFinalization) {
+        const derivedDbDisputeInfo = await db.findDisputeDocs({
```

Comment:
This could be one query outside of the loop to get all records for the markets in marketIds

Reply:
I think I tried that, but it was giving me an error when I tried to use `$in` in the query. Will try again to see what the exact error was.

Reply:
Actually I don't think I need to get the dispute docs, do I? I should be able to get the winning/tentative payout now without doing that.

Reply:
yup, true